functions/zoom-webhook.py

```diff
@@ -207,6 +207,10 @@ def construct_sqs_message(payload, context, zoom_event):
     if "on_demand_series_id" in payload:
         sqs_message["on_demand_series_id"] = payload["on_demand_series_id"]
 
+    # not used in downloader or uploader but useful for cloudwatch dashboard
```

Comment:
I want to have the zoom processing times, and the size of the file available in one log message.